### PR TITLE
Feature/#16 landlord routing

### LINKED ISSUE #16: Brainstorming the routing setup for the Landlord feature
Discuss among your group how best the routing for this feature can be implemented.

### PR DESCRIPTION
**Pull request by GiuliaReas**
# Feature/Change Description

 Added landlord-routing and landlord.module
- Added landlord/profile-routing and profile.module
- Added landlord/real estate-routing and real estate.module
- Added landlord/profile-container, landlord/profile-edit and landlord/profile-view
- Added landlord/real estate-container and landlord/real estate-list, landlord/real estate-add new, real estate-edit

# Developer Checklist

- [ ] Updated documentation or README.md
- [ ] If adding new feature(s), added and ran unit tests
- [ ] If create new release, bumped version number
- [x] Ran `npm run style:fix` for code style enforcement
- [x] Ran `npm run lint:fix` for linting
- [ ] Ran `npm audit` to discover vulnerabilities

### CODE REVIEW

File: src/app/modules/core/landlord/landlord-profile/components/landlord-profile-edit/landlord-profile-edit.component.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'cav-edit',
```
Comment (pc-codingavision) [inline src/app/modules/core/landlord/landlord-profile/components/landlord-profile-edit/landlord-profile-edit.component.ts]: Please modify the selector accordingly.

File: src/app/modules/core/landlord/landlord-profile/components/landlord-profile-view/landlord-profile-view.component.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'cav-view',
```
Comment (pc-codingavision) [inline src/app/modules/core/landlord/landlord-profile/components/landlord-profile-view/landlord-profile-view.component.ts]: Please modify the selector accordingly.

File: src/app/modules/core/landlord/landlord.module.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { CommonModule } from '@angular/common'
+import { NgModule } from '@angular/core'
+
+import { LandlordProfileModule } from './landlord-profile/landlord-profile.module'
+import { LandlordRoutingModule } from './landlord-routing.module'
+import { RealestateModule } from './realestate/realestate.module'
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, LandlordRoutingModule, LandlordProfileModule, RealestateModule],
```
Comment (pc-codingavision) [inline src/app/modules/core/landlord/landlord.module.ts]: Modules have been configured as lazy so they shouldn't be imported.

File: src/app/modules/core/landlord/realestate/realestate-routing.module.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  {
+    path: 'edit',
+    component: RealestateEditComponent,
+  },
+  {
+    path: 'list',
+    component: RealestateListComponent,
+  },
+  {
+    path: 'add-new',
```
Comment (pc-codingavision) [inline src/app/modules/core/landlord/realestate/realestate-routing.module.ts]: Please change to `new`.

**Review by pc-codingavision — CHANGES_REQUESTED** [review]

**Comment by GiuliaReas** [comment]
modifiche effettuate

**Pull request merged**